app/src/main/java/com/willowtree/vocable/settings/EditKeyboardFragment.kt

```diff
@@ -100,7 +100,7 @@ class EditKeyboardFragment : BaseFragment() {
         super.onViewCreated(view, savedInstanceState)
 
         binding?.backButton?.action = {
-            if (binding?.keyboardInput?.text.toString() == phrase?.localizedUtterance?.get(Locale.US.language) || arguments?.getBoolean(
+            if (binding?.keyboardInput?.text.toString() == phrase?.getLocalizedText() || arguments?.getBoolean(
```

Comment:
It's a lil hard to make our what this boolean is checking for. Additionally, maybe these conditions can be leveraged out elsewhere so that there isn't too much logic in the Fragment? Idk if a helper class or viewmodel can get the fragment the info it needs, not too much of a worry for me but something to consider.

Reply:
I pulled stuff out into variables so it's more clear 👍